test/integration/change-streams/change_stream.test.ts

```diff
@@ -1742,6 +1742,39 @@ describe('Change Streams', function () {
       });
     });
   });
+
+  it(
+    'for await loops should exit with error when changeStream closes',
```

Comment:
let's make sure we're setting a good example for test case descriptions: `"for await loops"` is what you are describing, hence `describe` is the appropriate block; `it` starts with a verb and contains the behavior

Reply:
I moved the test into the iterator api section under a `#close()` section